<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-5DBSC07H" failures="1" tests="3" name="Login" time="19.228" errors="0" timestamp="2022-10-11T15:52:58 IST">
  <testcase classname="testCases.LoginTest" name="login" time="1.937"/>
  <testcase classname="testCases.LoginTest" name="login" time="1.462"/>
  <testcase classname="testCases.LoginTest" name="logintest" time="1.883">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at testCases.LoginTest.logintest(LoginTest.java:54)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- logintest -->
</testsuite> <!-- Login -->
